packages/ui/primitives/document-flow/add-subject.tsx

```diff
@@ -105,9 +131,9 @@ export const AddSubjectFormPartial = ({
                 </li>
                 <li className="text-muted-foreground">
                   <code className="text-muted-foreground bg-muted-foreground/20 rounded p-1 text-sm">
-                    {'{signer.email}'}
+                    {'{signer.meta}'}
                   </code>{' '}
-                  - The signer's email
+                  - The signer's meta
```

Comment:
Caught you using Ctrl+F for find and replace 😆 

Rename symbol works pretty well btw.

Reply:
Oops! I missed this, lol. Thanks for pointing it out, will send a fix :)